docs/how-to-guides/sparksql-source-notes.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Storing secrets in Azure Key Vault
```

Comment:
Please fix title

Reply:
Changed title to 'Using `SparkSQLSource` as Data Source'